src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -76,4 +72,15 @@ public void toQuestion(Question question) {
     public String toString() {
         return "Answer [id=" + getId() + ", writer=" + writer + ", contents=" + contents + "]";
     }
+
+    public void validateForDelete(NsUser loginUser) throws CannotDeleteException {
+        if (!isOwner(loginUser)) {
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
```

Comment:
```suggestion
            throw new CannotDeleteException("...");
```

`Answer`입장에서는 다른 메시지가 적합하지 않을까요? 😄